newrelic/resource_newrelic_alert_policy_channel.go

```diff
@@ -26,7 +26,7 @@ func resourceNewRelicAlertPolicyChannel() *schema.Resource {
 				Description: "The ID of the policy.",
 			},
 			"channel_ids": {
-				Type:        schema.TypeList,
+				Type:        schema.TypeSet,
```

Comment:
Thought this might have to be a migration, but appears from testing that the implementation of Set vs List are compatible in the state file.

Reply:
Compatible in the state file yes, but the behavior is different. Any chance we need a migrate state function? - e.g. basically would be the same as `migrateStateNewRelicDashboardV0toV1` that we already have in Dashboards?

Reply:
I wouldn't say zero chance, though testing didn't break anything.  Probably best to be safe and implement it that way (though `migrateStateNewRelicDashboardV0toV1` is a no-op, so not sure the difference besides explicitly calling out that the schema changed, which is valid